.gitignore

```diff
@@ -9,6 +9,5 @@
 **/deployments
 .devcontainer
 packages/subgraph/generated
-packages/app/contracts/abi/*.json
```

Comment:
I think we want this in order to not have the abis in the repo, because the abis are generated from the contracts on when we run `yarn build` f from the project root. In order to always have the contracts, abis and the app in sync